doc/release/upcoming_changes/17116.expired.rst

```diff
@@ -0,0 +1,2 @@
+* The 14-year deprecation of ``np.ctypeslib.ctypes_load_library`` is expired.
+  Use :func:`~numpy.ctypeslib.load_library` instead, which is identical.
```

Comment:
Note that we seem to have two different styles of `.expired` notes - bullets and headers. I'm inclined to leave unifying them to when we release.

Reply:
Yeah, we can see that later, a mix may even be OK as well (I am not sure how long the bullet ones are). I was just looking if I ignore Travis here, and I think I will, its bogged down for ages.